java-spiffe-core/src/test/java/io/spiffe/workloadapi/FakeWorkloadApi.java

```diff
@@ -143,9 +157,17 @@ public void fetchJWTSVID(Workload.JWTSVIDRequest request, StreamObserver<Workloa
                 .setSvid(extraToken)
                 .build();
 
+        // This svid should be filtered out by the client because it has a non-unique hint.
```

Comment:
```suggestion
        // This JWT-SVID should be filtered out by the client because it has a non-unique hint and is not the first JWT-SVID in the response with this hint.
```